kong/dao/schemas_validation.lua

```diff
@@ -127,7 +127,9 @@ function _M.validate_entity(tbl, schema, options)
           local is_valid_type
           -- ALIASES: number, timestamp, boolean and array can be passed as strings and will be converted
           if type(t[column]) == "string" then
-            t[column] = utils.strip(t[column])
+            if schema.fields[column].trim_whitespace ~= false then
```

Comment:
We would also need unit tests or two in the plugins' validator test suite for this new field option, please. Especially since we use a `false` comparison vs. falsy (which is a good call)!